### PR TITLE
docs: add how-to videos

### PR DESCRIPTION
**Pull request by arbrandes**
Add the how-to videos from the [Instructional Series for Course Authors playlist](https://www.youtube.com/playlist?list=PL87xhvJSz2W5AhGcra8ojmxqIGSUetiZY).

### CODE REVIEW

File: source/educators/how-tos/add_course_updates.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Add Course Updates and Hand-outs
+###########################################
```
Comment (sarina) [inline source/educators/how-tos/add_course_updates.rst:2]: Nit: These should be the same length as the above title. Don't need to go about fixing all, but don't add more.

File: source/educators/how-tos/import_scorm_content.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Import SCORM Content
```
Comment (sarina) [inline source/educators/how-tos/import_scorm_content.rst:1]:
```suggestion
Importing SCORM Content
```

**Review by sarina — APPROVED** [review]

File: source/educators/how-tos/add_course_updates.rst (hunk 1 above)
Comment (arbrandes) [reply source/educators/how-tos/add_course_updates.rst]: +1.  I figure I'll fix the ones I've touched, at least.

**Comment by feanil** [comment]
Adolfo's out this week but I think this is good to land so I'm gonna merge it.

**Comment by ehuthmacher** [comment]
@feanil do you know when the videos will be linked to the correlating documentation on docs.openedx.org ?

**Comment by feanil** [comment]
@ehuthmacher this change landed a while ago, so I believe the videos should already be there.  Are you seeing instances where that is not the case? If so, can you provide some examples?

**Comment by ehuthmacher** [comment]
Ok great, thank you Feanil. I see it on the site. I was not aware that it has been integrated already. Thank you for this update.

**Pull request merged**